lib/constructs/attribute.js

```diff
@@ -118,10 +118,12 @@ class Attribute {
 
     if (!this.static && this.idl.special === "stringifier") {
       addMethod("toString", [], `
-        if (!this || !exports.is(this)) {
+        const esValue = this;
```

Comment:
This is correct per spec: <https://heycam.github.io/webidl/#es-stringifier>, <https://github.com/heycam/webidl/issues/855>.